README.md

```diff
@@ -0,0 +1,86 @@
+# Ansys actions
+
+[![ansys][Ansys badge]][actions.docs.ansys.com]
+[![CI-CD][CI-CD badge]][CI-CD yml]
+[![MIT][MIT badge]][MIT url]
+
+A repository containing a collection of [GitHub Workflows] to be reused by
+projects in the Ansys ecosystem.
+
+For more information on available actions and how to use them, see
+[actions.docs.ansys.com].
+
+## Recommended workflow strategy
+
+The different actions provided by [ansys/actions] can be used to create a
+simple but effective workflow.
+
+The following lines describe the suggested workflows for various events, such as
+pushing a new commit to a pull request, merging a commit to the main branch of a
+repository, and performing a new release. Each image showcases the distinct job
+steps declared in a YML file. Jobs highlighted in green signify that they
+execute, while those with a grey background indicate that they do not execute.
+
+For additional in-depth information refer to the poster [CI/CD pipelines for
+scientists].
+
+### Recommended workflow when pushing a new commit to a pull-request
+
+This workflow is recommended to ensure that the code ready to be merged is
+compliant with the project style, its code integrity, and that it is capable of
+successfully generating all the desired library artifacts.
+
+```mermaid
+graph LR
+    A(Doc style) --> B(Doc build)
+    C(Code style) --> D(Tests)
+    B & D --> E(Build library) --> F(Release library) & H(Doc deploy dev)
+    F --> G(Doc deploy stable)
+
+    classDef green fill:#d7e8d3, fill-opacity:0.5, stroke-width:2
```

Comment:
```suggestion
    classDef green fill:#d7e8d3, fill-opacity:0.5, stroke:#000000, stroke-width:2
```
I would also suggest adding a black border to avoid the purple color, @jorgepiloto , feel free to disregard if it's not necessary

Reply:
FYI, in dark mode the borders are not purple.
![image](https://github.com/user-attachments/assets/76c20c65-929e-44f6-ab0f-211aca2df328)